web-client/src/ducks/timeline/functions.ts

```diff
@@ -1,15 +1,114 @@
 import { firestore } from 'src/firebase';
```

Comment:
Added functions for querying timelines and offers for a single post.  For good measure, I also requery the request in case it has changed.